src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5428,17 +5428,17 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_NullabilityMismatchInParameterTypeOnExplicitImplementation_Title" xml:space="preserve">
     <value>Nullability of reference types in type of parameter doesn't match implemented member.</value>
   </data>
-  <data name="WRN_NullabilityMismatchInReturnTypeOnExplicitImplementationBecauseOfAttributes" xml:space="preserve">
-    <value>Nullability of reference types in return type doesn't match implemented member '{0}' because of nullability attributes.</value>
+  <data name="WRN_TopLevelNullabilityMismatchInReturnTypeOnExplicitImplementation" xml:space="preserve">
+    <value>Top-level nullability of reference types in return type doesn't match implemented member '{0}' (possibly because of nullability attributes).</value>
```

Comment:
>Top-level nullability [](start = 11, length = 21)

Consider removing the "top-level" qualifier from these warning messages.

"Top-level nullability" is probably not a term users are familiar with, and in most cases top-level nullability is the only nullability. Even in cases where there is top-level and nested nullability, the warning messages are simply indicating a nullability mismatch so it should be sufficient to say "nullability".